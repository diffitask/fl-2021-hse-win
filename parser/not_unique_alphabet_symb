Alphabet: 2 -- ("b", "b")
Q: 4
start: 0
runoff: 3
T: 2 -- 1, 3
edges: (0; 1; "b"), (0; 2; "b"), (1; 1; "b", "b"), (2; 3; "b", "b"), (3; 3; "b", "b")