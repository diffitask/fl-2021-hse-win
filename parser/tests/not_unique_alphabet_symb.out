Error: Symbols of the alphabet are not unique -- identical symbols: b (pos: 0) and b (pos: 1)
